default.nix

```diff
@@ -33,7 +33,11 @@ let
         };
       };
     };
-    daedalus = self.callPackage ./installers/nix/linux.nix {};
+
+    daedalus = self.callPackage ./installers/nix/linux.nix {
+      installer-generator = import ./installers/default.nix;
```

Comment:
if you just put installer-generator on the next line down, it will automatically be available to all of these packages

Reply:
Done.